packages/core/__tests__/cli/build.test.ts

```diff
@@ -466,8 +466,13 @@ describe('CLI: single-pass build mode typechecking', () => {
               2 const A = 2 * C;
                               ~
 
+              src/index.gts:2:15 - error TS2307: Cannot find module '@glint-test/a' or its corresponding type declarations.
```

Comment:
Since the TS/Volar upgrade, there seems to be some internal changes in the build algorithm, producing more error messages than before if one of the subprojects fails to build.